README.md

```diff
@@ -20,3 +20,11 @@ Download the latest release from [Tags] page and install the plugin using the
 1. Create an Incoming Webhook on slack: https://api.slack.com/messaging/webhooks#create_a_webhook
 
 2. Enter the webhook into the slacknotifier configuration section in DokuWiki's Configuration Settings
+
+## Root Namespace
+
+To incldue the root namespace, simply put a : in the namespace field in the config.
+
+## Discord
+
+This plugin will work for discord as well, if you append /slack to the discord webhook url; see https://birdie0.github.io/discord-webhooks-guide/other/slack_formatting.html 
```

Comment:
unrelated change, put it in a separate PR

Reply:
It is an observation related to the overall functionality. Besides, a I understand it, I would have to create a new fork to submit a separate PR. (As you an tell, I am not an expert on git; just trying to fix things)

Reply:
the change does not belong to this PR. this pr doesn't even describe such change. and no, you do not need another fork, just another PR. if you edit file via github web, it will offer you to make pr after making commit. just be sure to start editing the file in this repository not your fork.

ps github is not git. in fact you can make the changes without even using git, only github web.

Reply:
I'm sorry but I have no idea how to back out a single commit from the PR.

Reply:
If the change is merged via other pull request then the change won't be visible from this PR anymore. 

I will use squash merge anyway in this PR, as the commits are not that well authored (commit messages are useless) to be merged as single commits.

in that sense I'll reverse edit end remove the change with another commit.

Reply:
[`0a8fe8b` (#30)](https://github.com/glensc/dokuwiki-plugin-slacknotifier/pull/30/commits/0a8fe8ba5019969a70b63b6e437c647957742008)